engineering/ember.md

```diff
@@ -1 +1,11 @@
 # Ember Style Guide
+
+## Test
+
+### Test selector
+
+We try to use page objects as much as possible, but if we need to use test-selector directly in the test, import it using the following syntax:
```

Comment:
I'm not sure this is directly related to page objects. I think we can just say that we use test selectors for anything we need to reference in the DOM rather than css classes.